test/bin/common.sh

```diff
@@ -387,6 +387,7 @@ MICROSHIFT_Y1_OPTIONAL_RPMS_LIST=(
 MICROSHIFT_OPTIONAL_RPMS_LIST=(
     "${MICROSHIFT_Y1_OPTIONAL_RPMS_LIST[@]}"
     microshift-cert-manager
+    microshift-sriov
```

Comment:
```suggestion
    microshift-sriov
    microshift-sriov-release-info
```

Reply:
Oops you are right, was misled by cert-manager not having it, but thats actually a bug. Sorry @vanhalenar, we need to put it back, it was my bad.